app/forms/mappings/Formatters.scala

```diff
@@ -60,7 +60,7 @@ trait Formatters {
     override def bind(key: String, data: Map[String, String]): Either[Seq[FormError], String] =
       data.get(key) match {
         case None | Some("") => Left(Seq(FormError(key, errorKey)))
-        case Some(s) => Right(s)
+        case Some(s) => Right(s.trim.replace("‘", "'").replace("’", "'"))
```

Comment:
Unless you only want 2 occurrences changed 
Change s.trim.replace("‘", "'").replace("’", "'")   -> s.trim.replaceAll("‘", "'")